### PR TITLE
Update document the way to install optional extension

### PR DESCRIPTION
**Pull request by scalopus**

### CODE REVIEW

**Comment by rogeriopradoj** [comment]
Looks good to me. @CHH @loicfrering any thoughts?

**Comment by fabiang** [comment]
:+1:

**Comment by fabiang** [comment]
Could you please explain the `<version>` constraint how I did in #248?

**Comment by scalopus** [comment]
@fabiang PHPBUILD_INSTALL_EXTENSION="uprofiler=@" is that mean install from master branch?

**Comment by fabiang** [comment]
@scalopus: Yes. I haven't tested it but when you set `@mybranch` there, it should checkout `mybranch`.

**Comment by scalopus** [comment]
@fabiang I have updated document by following information in #248 . Please help to review.

**Comment by fabiang** [comment]
@scalopus @rogeriopradoj LGTM

**Comment by rogeriopradoj** [comment]
@scalopus Could you just rebase(or commit) placing all dates like March 2015? Thanks!

**Comment by scalopus** [comment]
@rogeriopradoj done.

File: man/php-build.5.html [hunk 1]
```diff
@@ -186,7 +186,7 @@ <h3 id="XDEBUG">XDEBUG</h3>
 
   <ol class='man-decor man-foot man foot'>
     <li class='tl'>php-build </li>
-    <li class='tc'>December 2012</li>
+    <li class='tc'>February 2015</li>
```
Comment (rogeriopradoj) [inline man/php-build.5.html]: -> March

File: man/php-build.5 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "PHP\-BUILD" "5" "December 2012" "php-build " "php-build"
+.TH "PHP\-BUILD" "5" "February 2015" "php-build " "php-build"
```
Comment (rogeriopradoj) [inline man/php-build.5]: March

**Comment by scalopus** [comment]
@rogeriopradoj those actually it just generated from tool (Ronn). I don't think that those should update manually. // I already updated it.

**Comment by rogeriopradoj** [comment]
Alright! Merged. Thanks, @scalopus !

**Pull request merged**